libraries/load-themed-styles/src/index.ts

```diff
@@ -7,14 +7,23 @@
 // entire node.js d.ts for now)
 declare var global: any; // tslint:disable-line:no-any
 
+/**
+ * @public
+ */
 export interface IThemingInstruction {
   theme?: string;
   defaultValue?: string;
   rawString?: string;
 }
 
+/**
+ * @public
+ */
 export type ThemableArray = Array<IThemingInstruction>;
```

Comment:
>ThemableArray [](start = 12, length = 13)

Why is this exported as a named type?  It would be better to just call it IThemingInstruction[] or ReadonlyArray<IThemingInstruction>